stoservice/src/main/java/com/tutrit/stoservice/controller/Request.java

```diff
@@ -1,10 +1,26 @@
 package com.tutrit.stoservice.controller;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Request {
     private String command;
+    private List<String> information = new ArrayList<>();
 
-    public Request(String command) {
-        this.command = command;
+    public Request(String inputString) {
+        String[] inputData = inputString.split(" ");
+        command = inputData[0];
+        for (int i = 1; i < inputData.length; i++) {
+            information.add(inputData[i]);
+        }
```

Comment:
If the user specifies a year instead of a model, can your code understand it?

Reply:
No, after the command you need to enter in the correct sequence.